### ISSUE TITLE
Importing YT videos and encoding takes hours

### ISSUE THREAD

**Issue by guymass** [WWBN/AVideo#6706]
I have pulled a YouTube video which is 720p about two hours. This took my encoder more than 5 hours to download and encode the video. This was much faster before plus the download takes place directly from YT server to my server which should be much faster. All in all something is not working right and this slowness causes the encoder to get filled and encode for many hours on some files. 

Files which are uploaded get encoded just fine, but those I get from other sites are going through a download+encoding process which take way too much time IMO.

This download took about five hours to complete:

![Screen Shot 5782-10-06 at 0 50 02](https://user-images.githubusercontent.com/58882903/172026631-2616db9c-d94e-45dc-a3bb-ef0236718a80.jpg)

**Comment by GreenDragonTavern** [comment]
https://github.com/WWBN/AVideo/issues/5972 might help.

**Comment by walt93** [comment]
There is another way, a different download script. Daniel has added support for it. See here: https://github.com/WWBN/AVideo/issues/5972#issuecomment-991991452

**Comment by guymass** [comment]
Great much thanks I have installed yt-dlp and testing now. The download was fast and encoding seems to start and appear as well on the encoder. I will close this once I see the file was created.

One other thing just thought to point out here is that we used to view the encoding process also on the videos page but now this feature is gone and I think is  a shame cause it was nice to see the encoding process also on the video page as well.

Now we can only see a message indicating it is "encoding":

![Screen Shot 5782-10-06 at 9 33 49](https://user-images.githubusercontent.com/58882903/172038643-05b0946f-7c0f-415d-b5b1-01700ecbdb91.jpg)
